asab/storage/elasticsearch.py

```diff
@@ -37,48 +41,41 @@ def __init__(self, app, service_name, config_section_name='asab:storage'):
 		super().__init__(app, service_name)
 		self.Loop = app.Loop
 
-		self.URL = asab.Config.get(config_section_name, 'elasticsearch_url')
-		# self._timeout = asab.Config.get(config_section_name, 'elasticsearch_timeout')
-
-		username = asab.Config.get(config_section_name, 'elasticsearch_username')
-		password = asab.Config.get(config_section_name, 'elasticsearch_password')
+		self.URL = Config.get(config_section_name, 'elasticsearch_url')
+		
+		parsed_url = urllib.parse.urlparse(self.URL)
+		self.ServerUrls = [
+			urllib.parse.urlunparse((parsed_url.scheme, netloc, parsed_url.path, None, None, None))
+			for netloc in parsed_url.netloc.split(',')
+		]
 
-		self.Refresh = asab.Config.get(config_section_name, 'refresh')
+		self.Refresh = Config.get(config_section_name, 'refresh')
 
+		username = Config.get(config_section_name, 'elasticsearch_username')
 		if username == '':
 			self._auth = None
 		else:
+			password = Config.get(config_section_name, 'elasticsearch_password')
 			self._auth = aiohttp.BasicAuth(login=username, password=password)
 
 		self._ClientSession = None
-		# get the first server
-		self.ESURL = self.get_servers_urls()
```

Comment:
@ateska Some components like Watcher still use the parsed ESURL from here - meaning, that after this change, review of other code will be needed.

Reply:
And what is ESURL meaning? It is misconception in the first place.

Reply:
https://github.com/TeskaLabs/asab/pull/180/commits/b625094af3003a85acc64901c829bf0d7369f6f2